src/DatePicker/Calendar.js

```diff
@@ -229,122 +226,156 @@ class Calendar extends React.Component {
     }
   };
 
+  yearSelector() {
+    if (!this.props.disableYearSelection) return (
+      <CalendarYear
+        key={'years'}
+        displayDate={this.state.displayDate}
+        onTouchTapYear={this.handleTouchTapYear}
+        selectedDate={this.state.selectedDate}
+        minDate={this.props.minDate}
+        maxDate={this.props.maxDate}
+      />
+    );
+  }
+
   render() {
     const {prepareStyles} = this.context.muiTheme;
-    const yearCount = yearDiff(this.props.maxDate, this.props.minDate) + 1;
+
+    /* const yearCount = yearDiff(this.props.maxDate, this.props.minDate) + 1;*/
     const weekCount = getWeekArray(this.state.displayDate, this.props.firstDayOfWeek).length;
     const toolbarInteractions = this.getToolbarInteractions();
     const isLandscape = this.props.mode === 'landscape';
+    const {calendarTextColor} = this.context.muiTheme.datePicker;
+
     const styles = {
       root: {
-        fontSize: 12,
+        color: calendarTextColor,
+      },
+      calendar: {
+        display: 'flex',
+        flexDirection: 'column',
       },
       calendarContainer: {
-        width: isLandscape ? 320 : '100%',
-        height: weekCount === 5 ? 284 :
-          weekCount === 6 ? 324 : 244,
-        float: isLandscape ? 'right' : 'none',
-        transition: transitions.easeOut('150ms', 'height'),
-        overflow: 'hidden',
+        alignContent: 'space-between',
+        display: 'flex',
+        flexDirection: 'column',
+        fontSize: 12,
+        fontWeight: 400,
+        height: 'auto',
+        justifyContent: 'space-between',
```

Comment:
Can we keep the flex properties (alignContent, flexDirection, justifyContent) next to each other in the object literal? For CSS grouping the properties is more useful than strict alphabetical order.